example/patches/react-native-vision-camera+2.16.6.patch

```diff
@@ -0,0 +1,65 @@
+diff --git a/node_modules/react-native-vision-camera/android/build.gradle b/node_modules/react-native-vision-camera/android/build.gradle
```

Comment:
Donated to vision camera - lets see if these get upstreamed and a new official release

Reply:
https://github.com/mrousavy/react-native-vision-camera/releases/tag/v2.16.7 - donated and its out